setup.cfg

```diff
@@ -20,17 +21,17 @@ setup-hooks =
 
 [files]
 data_files =
```

Comment:
Those data files placed under the expected paths if only the package was produced via setup.py directly, like:
```
PBR_VERSION=2.1.0.dev1006 python setup.py install bdist_rpm
```
But if built with ``rpmbuild -ba --define "%_topdir $PWD" contrib/packaging/rpm/kubespray.spec`` , all data files get copied under the /usr/local/share and I have no idea why! Here is the build log snippet:
```
+ /usr/bin/python2 setup.py build
running build
+ exit 0
Executing(%install): /bin/sh -e /var/tmp/rpm-tmp.wjmDCf
+ umask 022
+ cd /tmp/kubespray/BUILD
+ '[' /tmp/kubespray/BUILDROOT/kubespray-master-2.1.0.dev1006.x86_64 '!=' / ']'
+ rm -rf /tmp/kubespray/BUILDROOT/kubespray-master-2.1.0.dev1006.x86_64
++ dirname /tmp/kubespray/BUILDROOT/kubespray-master-2.1.0.dev1006.x86_64
+ mkdir -p /tmp/kubespray/BUILDROOT
+ mkdir /tmp/kubespray/BUILDROOT/kubespray-master-2.1.0.dev1006.x86_64
+ cd kubespray-master
+ export PBR_VERSION=2.1.0.dev1006
+ PBR_VERSION=2.1.0.dev1006
+ export SKIP_PIP_INSTALL=1
+ SKIP_PIP_INSTALL=1
+ /usr/bin/python2 setup.py install --skip-build --root /tmp/kubespray/BUILDROOT/kubespray-master-2.1.0.dev1006.x86_64
running install
running install_data
creating /tmp/kubespray/BUILDROOT/kubespray-master-2.1.0.dev1006.x86_64/usr
creating /tmp/kubespray/BUILDROOT/kubespray-master-2.1.0.dev1006.x86_64/usr/local
creating /tmp/kubespray/BUILDROOT/kubespray-master-2.1.0.dev1006.x86_64/usr/local/share
creating /tmp/kubespray/BUILDROOT/kubespray-master-2.1.0.dev1006.x86_64/usr/local/share/kubespray
creating /tmp/kubespray/BUILDROOT/kubespray-master-2.1.0.dev1006.x86_64/usr/local/share/kubespray/roles
creating /tmp/kubespray/BUILDROOT/kubespray-master-2.1.0.dev1006.x86_64/usr/local/share/kubespray/roles/rkt
creating /tmp/kubespray/BUILDROOT/kubespray-master-2.1.0.dev1006.x86_64/usr/local/share/kubespray/roles/rkt/defaults
copying roles/rkt/defaults/main.yml -> /tmp/kubespray/BUILDROOT/kubespray-master-2.1.0.dev1006.x86_64/usr/local/share/kubespray/roles/rkt/defaults
```

Reply:
Oh, I have to modify the build command it seems!